gap/NautyGraph.gi

```diff
@@ -373,23 +407,37 @@ BindGlobal( "NAUTYTRACESINTERFACE_IsomorphismGraphsOnlyEdges",
     
     return fail;
 end );
+BindGlobal( "NAUTYTRACESINTERFACE_IsomorphismGraphsOnlyEdges_List",
+  function( graph1, graph2 )
+    local can1, can2;
+
+    can1 := CanonicalForm( graph1 );
+    can2 := CanonicalForm( graph2 );
+    if can1!.edge_list = can2!.edge_list then 
+        return CanonicalLabelingInverse( graph1 ) * CanonicalLabeling( graph2 );
+    fi;
+    
+    return fail;
```

Comment:
The coverage failure here indicates that a test is missing which tests two non-isomorphic graphs for being isomorphic, I think?